lib/src/barcode_scanner.dart

```diff
@@ -43,6 +43,20 @@ abstract class BarcodeScanner {
 
 /// Controller interface for [BarcodeScanner]
 abstract class BarcodeScannerController {
+  /// Whether the device can switch between cameras (for example front vs back)
+  Future<bool> get supportsSwitchingCamera async => false;
+
+  bool get isSupported;
```

Comment:
isSupported is to generic. for me it is not clear what this actually does or why it is used